release/create_release_archives.sh

```diff
@@ -15,16 +15,12 @@
 #
 ################################################################################
 
-set -o errexit
```

Comment:
why do you move these down?

Reply:
```
set -o errexit
set -o pipefail
set -x
```
Move them together to ensure perform this cmd `create_release_archives.sh` only show the usage without unnecessary information

Before changes:
```
chunlins-mbp:istio clyang$ ./release/create_release_archives.sh 
++ mktemp -d /tmp/istio.version.XXXX
+ TEMP_DIR=/tmp/istio.version.wORR
+ BASE_DIR=/tmp/istio.version.wORR
+ ISTIOCTL_SUBDIR=istioctl
+ OUTPUT_PATH=
+ VER_STRING=
+ getopts d:i:o:v: arg
+ [[ -z /tmp/istio.version.wORR ]]
+ [[ -z '' ]]
+ usage
+ echo './release/create_release_archives.sh
    -d <path> path to use for temp directory                  (optional, randomized default is /tmp/istio.version.wORR )
    -o <path> path where build output/artifacts are stored    (required)
    -i <name> subdirectory in -o path to use for istioctl     (optional)
    -v <ver>  version info to include in filename (e.g., 1.0) (required)'
./release/create_release_archives.sh
    -d <path> path to use for temp directory                  (optional, randomized default is /tmp/istio.version.wORR )
    -o <path> path where build output/artifacts are stored    (required)
    -i <name> subdirectory in -o path to use for istioctl     (optional)
    -v <ver>  version info to include in filename (e.g., 1.0) (required)
+ exit 1
```
After Changes:
```
chunlins-mbp:istio clyang$ ./release/create_release_archives.sh 
./release/create_release_archives.sh
    -d <path> path to use for temp directory                  (optional, randomized default is /tmp/istio.version.XXXX )
    -o <path> path where build output/artifacts are stored    (required)
    -i <name> subdirectory in -o path to use for istioctl     (optional)
    -v <ver>  version info to include in filename (e.g., 1.0) (required)

```

Reply:
Let's ignore those for now. This file is not mean to be used by human directly anyway, and @rkpagadala is moving stuff around.

I suggest we just revert.

Reply:
@hklai If we can make it use by human directly, why not. At least, when I did test the release package, I need to run this Shell to generate the package. Then show the usage more friendly is not so bad. What do you think?

Reply:
OK it's fine to make it more usable.